chia/daemon/server.py

```diff
@@ -567,28 +592,15 @@ async def get_routes(self, websocket: WebSocketResponse, request: Dict[str, Any]
         return response
 
     async def get_wallet_addresses(self, websocket: WebSocketResponse, request: Dict[str, Any]) -> Dict[str, Any]:
-        all_keys = Keychain().get_keys(include_secrets=True)
         fingerprints = request.get("fingerprints", None)
```

Comment:
I suppose if the "fingerprints" value is not a `List[uin32]`, you get a poor error message. We assume that's the type in the call to `_get_keys_by_fingerprints()`

Reply:
Added assertion.